staging/src/k8s.io/kubectl/pkg/polymorphichelpers/objectrestarter.go

```diff
@@ -30,6 +30,7 @@ import (
 )
 
 func defaultObjectRestarter(obj runtime.Object) ([]byte, error) {
```

Comment:
Persisting data in the API as UTC seems reasonable to me, but I think we want to keep display of non-API output human-readable / in local time. Are these annotations displayed as-is or are they parsed somewhere else for display? Will this change make that display switch to UTC?

Reply:
These changes will be updated to Annotation, and after some consideration, I think this place should be rolled back and no changes should be made

Reply:
> Persisting data in the API as UTC seems reasonable to me, but I think we want to keep display of non-API output human-readable / in local time. Are these annotations displayed as-is or are they parsed somewhere else for display? Will this change make that display switch to UTC?

I seem to have misunderstood what you meant. I'm going to split it into two parts, one just to unify the timezone in the cli. The other part will be used to unify the time zone in the API data (this needs to be done very carefully)